TFNetworkRecLayer.py

```diff
@@ -3049,7 +3050,7 @@ def _get_cell(self):
       return self._cell
     rec_layer = self.network.parent_layer
     assert isinstance(rec_layer, RecLayer)
-    cell = self.network.parent_layer
+    cell = self.network.parent_layer.cell
```

Comment:
Whoops. How was it actually possible that we/I did not run into this bug before?

Reply:
We always set the cell when creating a TemplateLayer in `GetLayer.__call__()`, such that `self._cell` is returned. I didn't do that in `_TemplateLayer.get_sub_layer` (I guess I should/could have...).